cpp/test/CMakeLists.txt

```diff
@@ -4,23 +4,18 @@ project(dolfinx-tests LANGUAGES C CXX)
 
 if (PROJECT_IS_TOP_LEVEL)
   include(CTest) # enables testing
-  find_package(DOLFINX REQUIRED)
-endif()
 
-find_package(Catch2 3)
+  set(CMAKE_C_STANDARD 17) # For FFCx generated .c files.
```

Comment:
Is this necessary? Modern cmake specifies these on the target.

Reply:
Maybe not - it was there before.